client/components/ConfirmDialog.vue

```diff
@@ -3,7 +3,7 @@
 		<div v-if="data !== null" id="confirm-dialog">
 			<div class="confirm-text">
 				<div class="confirm-text-title">{{ data.title }}</div>
-				<p>{{ data.text }}</p>
+				<p v-html="data.text"></p>
```

Comment:
With this you introduce a security failure... data.text contains network.name which is user controlled.
Meaning this enables arbitrary injection of html with all that entails.
You should never use v-html for user controlled content